src/core/GCAudio.qml

```diff
@@ -88,11 +100,25 @@ Item {
      */
     signal done
 
+    property var muteChangeHandler: function () { muted ? volume = 0 : volume = 1 }
+
     /**
      *  When mute is changed we set the volume to 0 to mute a potential playing
      * sound.
      */
-    onMutedChanged: muted ? audio.volume = 0 : audio.volume = 1
+    onMutedChanged: muteChangeHandler()
+
+    //Pauses the currently playing audio
+    function pause() {
+        if(playbackState === Audio.PlayingState)
+            audio.pause()
+    }
+
+    //Resumes the current audio if it had been paused
+    function resume() {
+        if(playbackState === Audio.PausedState)
```

Comment:
I would start the music even if it is stopped

Reply:
Done :+1: